optimade/server/routers/utils.py

```diff
@@ -157,9 +157,14 @@ def get_entries(
         query = urllib.parse.parse_qs(parse_result.query)
         query["page_offset"] = int(query.get("page_offset", [0])[0]) + len(results)
         urlencoded = urllib.parse.urlencode(query, doseq=True)
-        links = ToplevelLinks(
-            next=f"{parse_result.scheme}://{parse_result.netloc}{parse_result.path}?{urlencoded}"
-        )
+        if CONFIG.base_url:
+            if parse_result.netloc.split("/")[1:]:
+                base_url = f"{CONFIG.base_url}/{parse_result.netloc.split('/')[1:]}"
```

Comment:
can you add a comment on what this does?

Reply:
Looked into what `netloc` actually is, and found that I don't need to do this, so condensed it to either be `CONFIG.base_url` or based on the request.